jobs/update-colin-filings/update_colin_filings.py

```diff
@@ -90,26 +74,19 @@ def send_filing(app: Flask = None, filing: dict = None, filing_id: str = None):
     """Post to colin-api with filing."""
     clean_none(filing)
 
-    # validate schema
```

Comment:
Add a comment to your PR's description of changes calling this out (for visibility).